cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -907,6 +909,25 @@ pPoolMetaDataFile =
       <> Opt.completer (Opt.bashCompleter "file")
       )
 
+pTxMetadataJsonSchema :: Parser TxMetadataJsonSchema
+pTxMetadataJsonSchema =
+    (  Opt.flag' ()
+        (  Opt.long "--json-metadata-no-schema"
```

Comment:
I guess that two hyphens in `Opt.long "--json-metadata-no-schema"` and on line 922 `Opt.long "--json-metadata-detailed-schema"` are not needed.